src/sentry/static/sentry/app/routes.jsx

```diff
@@ -239,6 +239,22 @@ const projectSettingsRoutes = [
       import(/*webpackChunkName: "ProjectTeams"*/ './views/settings/project/projectTeams')}
     component={errorHandler(LazyLoad)}
   />,
+  <Route
```

Comment:
this seems like an awfully verbose way of declaring routes

Reply:
Yeah, we can now get rid of `key` by using `React.Fragment`, and create a `LazyRoute`, but we use `name` for breadcrumbs, and dynamic imports require static strings